lib/BackbeatConsumer.js

```diff
@@ -4,6 +4,7 @@ const kafkaLogging = require('kafka-node/logging');
 const async = require('async');
 const joi = require('joi');
 
+const BackbeatProducer = require('./BackbeatProducer');
```

Comment:
It would be ideal to keep `BackbeatProducer` and `BackbeatConsumer` separate, without requiring them in each other. We can make sure they pass tests independently. For bootstrap, since it's only for testing, you can pass producer object as a parameter.
Adding this comment here so that we can cleanup sometime.